app/models/user.rb

```diff
@@ -40,6 +40,7 @@ class User < ApplicationRecord
           :recoverable,
           :rememberable,
           :trackable,
+          :timeoutable,
```

Comment:
Is the timeout limit configured somewhere?

Reply:
yes in the devise.rb file